### PR TITLE
doc: refine installation documents

### PR DESCRIPTION
**Pull request by tokers**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues

___
### Bugfix
- Description

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

___
### Backport patches
- Why need to backport?

- Source branch

- Related commits and pull requests

- Target branch

### CODE REVIEW

**Comment by tokers** [comment]
@membphis @moonming @gxthrj Need your help to read these documents.

**Review by membphis — APPROVED** [review]

**Comment by membphis** [comment]
@tokers need to fix the conflict file

**Comment by tokers** [comment]
@membphis Fixed.

File: docs/samples/index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+# Ingress APISIX Use Examples
+
+This is an index page about use examples of Ingress APISIX. Read them if you're trying to learn Ingress APISIX.
+
+* [Proxy the httpbin service](#proxy-the-httpbin-service).
```
Comment (gxthrj) [inline docs/samples/index.md:24]: Should be 
[Proxy the httpbin service](./proxy-the-httpbin-service.md)

Comment (membphis) [reply docs/samples/index.md:24]: this way is better:

image

```suggestion
[Proxy the httpbin service](./proxy-the-httpbin-service.md)
```

Comment (gxthrj) [reply docs/samples/index.md:24]: Get it .

File: README.md [hunk 2]
```diff
@@ -117,10 +117,15 @@ We welcome all kinds of contributions from the open-source community, individual
...
-- ![Twitter Follow](https://img.shields.io/twitter/follow/ApacheAPISIX?style=social) - follow and interact with us using hashtag `#ApacheAPISIX`
-- [Bilibili video](https://space.bilibili.com/551921247)
+* Mailing List: Mail to dev-subscribe@apisix.apache.org, follow the reply to subscribe the mailing list.
+* QQ Group - 578997126
+* ![Twitter Follow](https://img.shields.io/twitter/follow/ApacheAPISIX?style=social) - follow and interact with us using hashtag `#ApacheAPISIX`
+* [Bilibili video](https://space.bilibili.com/551921247)
+
+## Terminology
+
+* Ingress APISIX: the whole service that contains the proxy (Apache APISIX) and ingress controller (apisix ingress controller).
```
[the remark this answers is not in the document]
Comment (tokers) [reply README.md]: Fixed.

File: docs/samples/index.md (hunk 1 above)
Comment (tokers) [reply docs/samples/index.md]: Changed.

**Review by gxthrj — APPROVED** [review]

**Pull request merged**